pom.xml

```diff
@@ -23,6 +23,7 @@
     <modules>
         <module>apps</module>
         <module>libs</module>
+        <module>apps/source-code-analyser</module>
```

Comment:
You don't need this @shivam-rawat-akto . 

You should add `source-code-analyser` in pom.xml of `apps` folder